tools/sql_extraction/build.gradle.kts

```diff
@@ -31,6 +38,7 @@ task<de.undercouch.gradle.tasks.download.Download>("downloadGrammars") {
     }
     src(listOf("https://raw.githubusercontent.com/antlr/grammars-v4/master/java/java9/Java9.g4"))
     dest("gen/main/antlr/")
+    overwrite(false)
     println(System.getProperty("user.dir"))
 }
 
```

Comment:
Please add some comments to list the commands on
- how to build
- how to run unit tests